packages/design-system/src/components/contextMenu/components/button.js

```diff
@@ -84,6 +85,9 @@ const Button = forwardRef(function Button(
     onFocus,
     forcePadding = false,
     dismissOnClick = true,
+    children,
```

Comment:
🤔  I don't think `children` needs to be destructured here since it'd be passed into the button through `props`
```suggestion
```

Reply:
Hm, it didn't work otherwise for the quick-action menu though.

Reply:
oh weird! Okay then ignore me here 😄